ERCS/erc-x.md

```diff
@@ -0,0 +1,275 @@
+---
+eip: Not Assigned
```

Comment:
```suggestion
eip: 7578
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
File name updated.

Reply:
@Milehigh.wor1d

@lucentlabz 

@

Reply:
![image](https://github.com/ethereum/ERCs/assets/167463404/4a4cd318-1347-4ab5-8ad4-360a0d207b78)